src/code-of-conduct.md

```diff
@@ -68,7 +68,7 @@ List of organisers:
 - Steve Messer
 - Tallulah Jackson-Marriott
 
-(These guidelines are adapted from the [Practical Service Design](http://www.practicalservicedesign.com/getting-started-on-slack/) community guidelines, [Afrotech Fest](https://www.afrotechfest.co.uk/coc/), [Code for America](http://www.cvent.com/events/code-for-america-summit-2018/custom-40-e12d85b157b94d69b80d8911cc641d36.aspx), [UKGovcamp](https://www.ukgovcamp.com/code-of-conduct/), and [Services Week 2021](https://docs.google.com/document/d/1vQchJh-s6Fu6F4bN8UFM4lvRChe8ERgsE5whf9pUfjA/edit?usp=sharing) codes of conduct.)
+(These guidelines are adapted from [Afrotech Fest](https://www.afrotechfest.co.uk/coc/), [Code for America](http://www.cvent.com/events/code-for-america-summit-2018/custom-40-e12d85b157b94d69b80d8911cc641d36.aspx), [UKGovcamp](https://www.ukgovcamp.com/code-of-conduct/), and [Services Week 2021](https://docs.google.com/document/d/1vQchJh-s6Fu6F4bN8UFM4lvRChe8ERgsE5whf9pUfjA/edit?usp=sharing) codes of conduct.)
```

Comment:
Yeah, this one's fine if the content we're giving credit to has been retired.

One option would be to [link to an archive page](https://web.archive.org/web/20220223040226/http://www.practicalservicedesign.com/getting-started-on-slack/) but it feels excessive.

Reply:
I agree I think that's a bit much. If you're happy I'll resolve this comment.